client/src/components/EditAssessment/EditAssessment.tsx

```diff
@@ -0,0 +1,216 @@
+import { useState } from "react";
+import { classnames } from "../../Utilities";
+import Button from "../Button";
+import blob from "./blob-small.svg";
+import { Assessment } from "../../logic/icsGen";
+
+interface EditAssessmentProps {
+  onClose: () => void;
+  onSave: (assessment: Assessment) => void;
+  assessment: Assessment;
+}
+
+const EditAssessment = ({
```

Comment:
I think... `EditAssessment` is more of a page than a component..? But let's leave it like this for now.